samtools-1.3/Makefile

```diff
@@ -0,0 +1,22 @@
+build_tool = runtime-container.DONE
+git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../samtools-1.3 | cut -f1 -d " ")
+name = quay.io/ucsc_cgl/samtools
```

Comment:
Judging from ubu and mapsplice, I think the image name should be just "samtools" and the version should be in the tag.